cucumber/api/features/step_definitions/authz_steps.rb

```diff
@@ -82,7 +82,7 @@
 end
 
 Given(/^I create a binary secret value?$/) do
-  @value = Random.new.bytes(16)
+  @value = Random.new.bytes(16).force_encoding(Encoding::UTF_8)
```

Comment:
Per my note in Slack, changing the actual comparison in the step gets:

```expected: "$)=b\xB1\xB9\x16H*\x03\xAF$\xBE\x86g\x8D"
           got: "$)=b\xB1\xB9\u0016H*\u0003\xAF$\xBE\x86g\x8D"```

Which means the encoding needs to be enforced one way or the other to actually compare the strings correctly.  I think this is okay considering the data is only being interpolated as UTF-8 for comparison purposes, but I could be wrong.

Thoughts?